demo/deploy-bookstore.sh

```diff
@@ -39,8 +40,10 @@ metadata:
 EOF
 
 if [ "$DEPLOY_ON_OPENSHIFT" = true ] ; then
-    oc adm policy add-scc-to-user privileged -z "$SVC" -n "$BOOKSTORE_NAMESPACE"
-    oc secrets link "$SVC" "$CTR_REGISTRY_CREDS_NAME" --for=pull -n "$BOOKSTORE_NAMESPACE"
+    oc adm policy add-scc-to-user privileged -z bookstore -n "$BOOKSTORE_NAMESPACE"
```

Comment:
These should still be $SVC instead of bookstore since this script is used to configure the  bookstore-v1 and bookstore-v2 service accounts

Reply:
```suggestion
    oc adm policy add-scc-to-user privileged -z "$SVC" -n "$BOOKSTORE_NAMESPACE"
```